src/main/java/dk/kb/util/webservice/stream/ContinuationStream.java

```diff
@@ -57,25 +57,41 @@ public ContinuationStream(Stream<T> inner, C continuationToken, Boolean hasMore)
      * @param continuationToken used for requesting a new stream that continues after the last element of the
      *                          current stream. If {@code null}, no continuation information is available.
      * @param hasMore           whether or not a subsequent request for a stream is likely to produce any elements.
+     * @param recordCount       the amount of records in the stream.
+     */
+    public ContinuationStream(Stream<T> inner, C continuationToken, Boolean hasMore, Long recordCount) {
+        this(inner, continuationToken, hasMore, recordCount, null);
+    }
+
+    /**
+     * Create a stream.
+     *
+     * @param inner             the provider of the elements.
+     * @param continuationToken used for requesting a new stream that continues after the last element of the
+     *                          current stream. If {@code null}, no continuation information is available.
+     * @param hasMore           whether or not a subsequent request for a stream is likely to produce any elements.
+     * @param recordCount       the amount of records in the stream. This can be {@code null}.
      * @param responseHeaders HTTP headers from the response that {@code inner} was constructed from.
      */
-    public ContinuationStream(Stream<T> inner, C continuationToken, Boolean hasMore,
+    public ContinuationStream(Stream<T> inner, C continuationToken, Boolean hasMore, Long recordCount,
```

Comment:
`kb-util` is used in multiple products, so changing the signature of `public` (or `protected`) methods might cause `kb-util`-using projects to fail to compile.

Please provide an overloaded version of the method that has the old signature.